src/Persistent/Implementation.hs

```diff
@@ -2,22 +2,29 @@
 
 {-# OPTIONS_GHC -Wall -Werror #-}
 
-module Persistent.Implementation (mkRunPersist, withPersist)
+module Persistent.Implementation (mkRunPersist, withPersist, withPersist')
 where
 
-import Control.Lens
 import Config
+import Control.Exception (finally)
+import Control.Lens
+import Data.Monoid
 import Persistent.Api
```

Comment:
Anyway travis is good at catching those.  I never read the import sections any more.  (-: